up-subscription/src/handlers/fetch_subscribers.rs

```diff
@@ -83,12 +85,26 @@ impl RequestHandler for FetchSubscribersRequestHandler {
         };
 
         // Build and return result
+        let (subscribers, has_more) = fetch_subscribers_response;
+        let mut subscriber_infos: Vec<SubscriberInfo> = vec![];
+        for subscriber in subscribers {
+            subscriber_infos.push(SubscriberInfo {
+                uri: Some(subscriber).into(),
+                ..Default::default()
+            });
+        }
```

Comment:
Just FYI, this should also work ...
```suggestion
        let subscriber_infos = subscribers.iter()
          .map(|subscriber| SubscriberInfo {
            uri: Some(subscriber).into(),
            ..Default::default()
          })
          .collect();
```

Reply:
I keep forgetting to use map()